### ISSUE TITLE
Feedback for “Examples of Prompts”

### ISSUE THREAD

**Issue by yashg505** [dair-ai/Prompt-Engineering-Guide#167]
I checked the prompt for "nutral" and asked ChatGPT if it deliberately ignored the word nutral or did it correct the word in the output.
Here's the response from AI:
"apologize for the confusion. I did not deliberately ignore the word "nutral" (which I assume was meant to be "neutral"). It was likely a typo or misspelling in your previous message. In my response, I used the correct spelling of "neutral"."

**Comment by yashg505** [comment]
![Screenshot from 2023-05-02 23-14-46](https://user-images.githubusercontent.com/94542636/235797432-59b9acd5-b4c6-4e2d-b5bb-56d2d2d039ca.png)

**Comment by omarsar** [comment]
Yeah. So this is the primary reason why we needed to use something like few-shot prompting as discussed in the prompting guide.
